CONTRIBUTING.rst

```diff
@@ -106,6 +106,17 @@ After talking to the Satpy developers any additional work like code or
 documentation changes can be provided as a GitHub
 `Pull Request <https://help.github.com/articles/creating-a-pull-request/>`_.
 
+To make sure that your code complies with the pytroll python standard, you can
+run the `flake8 <http://flake8.pycqa.org/en/latest/>`_ linter on you change
+before you submit it, or even better install a pre-commit hook that runs the
+style check for you. To this aim, we provide a configuration file for the
+`pre-commit <http://pre-commit.com>`_ tool, that you can install with eg::
+
+  pip install pre-commit
+  pre-commit install
+
+running for your base satpy directory.
```

Comment:
That has to be run "from" the base satpy directory? Could you also mention that this will automatically check code style for every commit? Or is that too redundant based on what you've said already.

Reply:
I can add that.
```suggestion
running from your base satpy directory. This will automatically check code style for every commit.
```